src/main/java/it/gov/pagopa/controller/consent/PortalConsentController.java

```diff
@@ -39,46 +35,38 @@ public interface PortalConsentController {
                     }),
                     @ApiResponse(responseCode = "500", description = "Server ERROR", content = {
                             @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorDTO.class))
-                    }),
-                    @ApiResponse(responseCode = "503", description = "Service Unavailable", content = {
-                            @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorDTO.class))
                     })
             }
 
     )
-    @RequestMapping(
-            method = RequestMethod.GET,
+    @GetMapping(
             value = "/consent",
             produces = { "application/json" }
     )
-    ResponseEntity<PortalConsentDTO> getPortalConsent(@RequestHeader(value = "Content-Language") String contentLanguage) throws JsonProcessingException;
+    ResponseEntity<PortalConsentDTO> getPortalConsent(@RequestParam(value = "uid") String userId) throws JsonProcessingException;
```

Comment:
the userId should be read from the session token..
it could be possible let it ASIS if this API is not exposed inside the APIM.

I will let @StefanoTerragni to decide about this